src/System.Private.Xml/tests/XmlTextReader/System.Xml.XmlTextReader.Tests/XmlTextReaderAttributeTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Xml.Tests
+{
+    public class XmlTextReaderAttributeTests
+    {
+        [Fact]
+        public void XmlTextReaderGetAttributeWithIndexTest()
+        {
+            XmlTextReader textReader = XmlTextReaderTestHelper.CreateReader("<element1 attr='val'> abc </element1>");            
+            Assert.True(textReader.Read());
+            Assert.Equal("val", textReader.GetAttribute(0));
+        }
+
+        [Fact]
+        public void XmlTextReaderMoveToAttributeWithNameTest()
+        {
+            XmlTextReader textReader = XmlTextReaderTestHelper.CreateReader("<element1 attr='val'> abc </element1>");
+            Assert.True(textReader.Read());
+            Assert.True(textReader.MoveToAttribute("attr"));
+            Assert.Equal(XmlNodeType.Attribute, textReader.NodeType);
+            Assert.Equal("val", textReader.Value);
+        }
+
+        [Fact]
+        public void XmlTextReaderMoveToAttributeWithNamespace()
```

Comment:
- two namespaces and 1 attribute in each namespace - check if chosen the correct one
- why is the NameTable argument needed here?